api/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "grow-roadmaps-api",
+  "version": "0.0.1",
+  "author": "Marcos Nunes",
+  "license": "MIT",
```

Comment:
Se sua proprosta for usar a licença MIT precisamos em seguida atualizar o [LICENSE.d](https://github.com/MarcosRenato266/Grow-Roadmaps/blob/master/LICENSE.md)

Reply:
Dei merge para se quiserem já irem fazendo essas atualizações, hoje vai ser complicado para mim